app/views/shared/_card-listings-l.html.erb

```diff
@@ -0,0 +1,20 @@
+<div class="card-listings-l">
+  <div class="picture">
+    <img src="#" alt="">
+    <div class="profile d-flex">
+      <%= image_tag "source", alt: "alttext" %>
+      <%= @user.name %>
```

Comment:
I managed to test the listing views while testing the routes :) ActionController raised an error here because we only have `first_name` or `last_name` attributes for `User` model. Could you please change `name` to either?

Reply:
I can change it, but where do i do that...

Reply:
If you return to the branch on your computer, is your code still there?

Reply:
yes, i was confused... i found them and updated! sorry...

Reply:
No problem!

Reply:
I think `@user` will need to be changed to `current_user` as well.